modules/quality/include/opencv2/quality/qualitybrisque.hpp

```diff
@@ -0,0 +1,81 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#ifndef OPENCV_QUALITY_QUALITYBRISQUE_HPP
+#define OPENCV_QUALITY_QUALITYBRISQUE_HPP
+
+#include "qualitybase.hpp"
+#include "opencv2/ml.hpp"
+
+namespace cv
+{
+namespace quality
+{
+
+/**
+@brief BRISQUE (Blind/Referenceless Image Spatial Quality Evaluator) is a type of No Reference
+Image Quality Assessment. It measures score based on extracting Natural Scene Satistics (https://en.wikipedia.org/wiki/Scene_statistics)
+and calculating feature vectors. @cite Mittal2 for original paper and @cite Mittal2_software for original implementation.
```

Comment:
Please add extra line after the first sentence to separate "brief" and "details" sections.

Sentence with citations can be improved.

Documentation preview for this patch is [here](http://pullrequest.opencv.org/buildbot/export/pr_contrib/2015/docs/d8/d99/classcv_1_1quality_1_1QualityBRISQUE.html)

Reply:
I agree this needs improvement.  That preview is very helpful, thanks for the link